bundles/org.openhab.binding.rotel/src/main/resources/OH-INF/thing/a14.xml

```diff
@@ -18,12 +18,15 @@
 			<channel id="treble" typeId="treble"/>
 			<channel id="frequency" typeId="frequency"/>
 			<channel id="brightness" typeId="brightness"/>
+			<channel id="tcbypass" typeId="tcbypass"/>
+			<channel id="balance" typeId="balance"/>
+			<channel id="speakera" typeId="speakera"/>
+			<channel id="speakerb" typeId="speakerb"/>
 		</channels>
 
 		<properties>
 			<property name="protocol">ASCII_V2</property>
 		</properties>
-
```

Comment:
Very minor but the blank line to separate properties an,d config was not bad.

Reply:
sorry - a mistake